src/libs/elektra/kdb.c

```diff
@@ -536,6 +540,13 @@ static int elektraGetDoUpdateWithGlobalHooks (KDB * handle, Split * split, KeySe
 	const int bypassedSplits = 1;
 	int pgs_done = 0;
 	int pgc_done = 0;
+
+	// GLOBAL: getstorage [init]
+	// GLOBAL: getstorage [max once]
+
+	// GLOBAL: postgetstorage [init]
+	// GLOBAL: postgetstorage [max once]
+
```

Comment:
just for your information: elektraGetDoUpdatewithGlobalHooks gets called twice (thats why there's the `run` parameter)